### ISSUE TITLE
Morris Bedding Removal Setup charges

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2427]
_Imported from trac ticket 2427.
Created by **dwarren**
Opened in trac: Jul 26, 2013, 4:29:43 PM
Last modified in trac: Oct 22, 2013, 8:41:44 AM_
***
_by dwarren:_
We would like to implement a charge for removing bedding as a Set up fee.  We are going to create a new sku to accomplish this, but it will not be entered on all bedding sales across the board.  The sales associate will need to enter the sku and then we would like the charge of $19.95 added as a setup fee.  Is there a way to prompt the user to either add or not add the removal sku whenever a certain sku is entered on the order?
The new sku is REMOVEBED.
Let me know if you need anything else from me on this.

Thanks.
Darlene

_____________________________________________
Darlene,

You will need to tell us how to identify the products you want flagged.  Then we can add REMOVEBED as a related item and we can also add a yellow warning that needs to be acknowledged like shown below.  Please address the questions below and I will get a ticket started for you:

 * How are the bedding products identified?---minor code 500 with a
   vendor code of SE11, SE13 or ST14
 * Do you want the warning or just a related item.---we like the way
   you have in the example below with a reminder that needs to be
   acknowledged on any order with items from above. **This is triggered
   by having information in the SALESPROMPTS field.  We need a more
   automated way of getting this to display than entering the same
   prompt on all the products above.**
 * Might you need to add multiple quantities of RemoveBed and have that
   reflected in the fee?---no, this will be a flat fee of $19.95 no
   matter how many items are on the order.
 * Is there somewhere in Product where you can store the fee?  This can
   be set up as a custom property.  This is better than hard coding the
   fee.---I have entered the price in the adv_prc field in ITM.

[[Image(dddeiiag.png)]]

***
* sjohnson attached [dddeiiag.png](https://trac.retailarchitects.com/trac/attachment/ticket/2427/dddeiiag.png) on Jul 26, 2013, 4:29:43 PM
* criley attached [ticket2427.patch](https://trac.retailarchitects.com/trac/attachment/ticket/2427/ticket2427.patch) on Aug 23, 2013, 12:05:21 PM

**Comment by kentbower** [comment]
_**sjohnson** trac comment on Jul 30, 2013, 10:47:54 AM:_

Summary:
 - Lets not mess with the reminders functionality through SALESPROMPTS.  Lets just go with a simple rule to warn in yellow where productclassid = '500' and vendor in ('SE11', 'SE13', 'ST14')
 - Get the Cost from the adv_prc field in ITM and put it in the Setup Charge field.  The quantity should not be more than 1.

**Comment by kentbower** [comment]
_**sjohnson** trac comment on Sep 9, 2013, 10:08:51 AM:_

Lowering priority, waiting on feedback from Morris.

**Comment by kentbower** [comment]
_**kbower** trac comment on Oct 22, 2013, 8:33:56 AM:_

related items added in [[4387]](../commit/97386c060b931d4da0326ac97c5851bae9caab5d)

**Comment by kentbower** [comment]
_**kbower** trac status change on Oct 22, 2013, 8:41:43 AM:_ **closed**

**Comment by kentbower** [comment]
_**kbower** trac comment on Oct 22, 2013, 8:41:44 AM:_

Closed as duplicate of [#2504](2504).
